backend/api/sessions.py

```diff
@@ -4,7 +4,7 @@
 
 
 def get_request_session(request: Request):
-    return request.session
+    return request.session  # pragma: no cover
```

Comment:
To [your comment](https://github.com/openstax/rope/pull/5#issuecomment-1932726331), I think this is fair to exclude from the coverage given our dependency injection in tests.